key-codepoint.cpp

```diff
@@ -0,0 +1,66 @@
+#include <stdlib.h>
```

Comment:
converting this to pure golang all that remains for this PR? am I understanding that correctly?

Reply:
I would also like to convert the Windows Ctrl key to MacOS Meta, for shortcuts, same with text selection..
I wand to remove the text related suff from go-flutter since it is implemented in the flutter framework now..
Since we are relying on GLFW, the flutter framework uses those key-presses to trigger Linux/Windows shortcuts. We need a small conversion layer to handle MacOS shortcuts.